### PR TITLE
Create communiteh-yes.tweet

### PR DESCRIPTION
**Pull request by zyler9985**

### CODE REVIEW

File: tweets/communiteh-yes.tweet [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+A new chapter to our story,
+And as we turn the page,
+Embrace the unexpected,
+For soon everything shall change” – Crypto Publius
+---
+“When I first came across Zenon, I was immediately struck by its branding. The sophistication and attention to detail are truly second to none. It made me feel like I was joining a platform that truly values professionalism.” – Layiid
+---
+“There was this unforgettable moment when Dogwithstyle responded instantly to Mr. Kaine with a BTC transfer. Little did we realize that we were witnessing a piece of Zenon's Network history with the signing of the Taproot block.” – Dogwithstyle
+---
+“I've seen firsthand how Zenon brings out the best in people. I've seen degens start with minimal knowledge and, through sheer dedication, rise to play significant roles within the community. It's a testament to Zenon's nurturing spirit.” – 0x3639
```
Comment (0x3639) [inline tweets/communiteh-yes.tweet]: Did I say this?  Maybe it was someone else?

**Review by 0x3639 — DISMISSED** [review]

**Comment by sol-znn** [comment]
Is it possible to have a thread of tweet links?

**Comment by 0x3639** [comment]
> Is it possible to have a thread of tweet links?

yes, you separate each tweet with

`---`

**Comment by 0x3639** [comment]
@mehowz what do you think about this tweet?

**Comment by coinselor** [comment]
Alien friends,

First and foremost, I'd like to express my sincere gratitude for entrusting me with the responsibility of reviewing content for our Twitter repo. It's an honor to be in this position, and I'm eager to contribute to Zenon Network's voice on such a significant platform.

While I understand the excitement and urgency to tweet and engage with our community, I believe it's paramount that we maintain the integrity and authenticity of our communication. Not utilizing this platform to its fullest potential can be perceived as a form of disrespect towards the trust placed in us by granting control of this vital communication channel. The keys to this channel are a privilege, and it's our duty to use them wisely and consistently. In line with this, I'd like to offer some constructive feedback on the original PR by @zyler9985:

1. **Incorporate Visuals**: As the saying goes, a picture is worth a thousand words. Including an image in the initial tweet of a thread can significantly boost its visibility, akin to the benefits of Twitter Blue. I'll take the initiative to liaise with our community members to source a fitting image.
  
2. **Tagging and Quoting**: Whenever we reference or share thoughts from other individuals, especially if they're on Twitter, it's essential to tag or quote tweet them. This not only gives due credit but also serves as a form of fact-checking. For content originating outside of Twitter, we might consider reaching out to the original authors and encouraging them to share their insights on Twitter. This approach will lend a more integrated and polished appearance to our thread.

3. **Optimizing Quotes**: Given that we don't have the luxury of extended character limits like Twitter Blue offers, we need to be judicious with our use of quotes. Leveraging quote tweets can be an effective workaround, allowing us to share more while staying within the constraints.

Lastly, I'd love to hear @mehowz's thoughts on this, but I believe incorporating a compelling Call-to-Action (CTA) is crucial. A warm invitation to be part of our journey, perhaps by joining our ecosystem, would be fitting. However, I'm aware that Twitter's algorithm isn't too kind to external links. As a workaround, we could extend the invitation and guide them to our twitter bio for more details.

I'm sure our content strategy and delivery will pick up momentum in no time.

**Review by sugoibtc — COMMENTED** [review]
"Being in the ZNN community has been an incredible journey so far, and I believe that the impact of NoM will be far greater than anyone could ever expect. Don't fade the momentum anon, keep stacking zats." - Sugoi

I would like to submit my part to the community tweet if still possible :)

**Comment by 0x3639** [comment]
Thank you for this thoughtful response.  One thing I would mention is adding links, images and RTs is not "simple" given how this framework works.  

Today, it's not possible for a submitter to add an image.  We need a /media folder in the root directory.  If we can get this PR approved, https://github.com/zenon-network/twitter/pull/13 I can then submit a PR to add a /media folder so we can insert images.  

@coinselor have a look at #13.  That might help people understand how to perform the actions you are recommending.  If we get #13 approved I will then submit a new PR so we can attach images.  

Maybe the first tweet should just be simple.  "Hello, this is Zenon Community"

**Comment by coinselor** [comment]
> @coinselor have a look at #13.

I had looked at it earlier today. It looked good to me. I wasn't showing as a reviewer yet so I didn't do anything. I just figured out I can request myself to review it. I'll approve the changes.

**Comment by zyler9985** [comment]
@coinselor thanks for the feedback! Your suggestions will definitely give communications a professional quality and polish to them. And I can't wait to see the image/pic the alien hive-mind generates.

As for quote-retweeting people to circumvent the character limit with a clean appearance, that's a good idea. We would need the following people to tweet so they can be quote retweeted within the thread: coinselor, sol sanctum and trevor.leazy. As for rex, he's taking a break from zenon and may be unreachable, but we can paraphrase to this which fits the character limit:
"My fave memory was when Apu was badgering Mr Kaine for testnet nft’s while Mr Kaine ignored him. Apu injected fun and energy into our community and made people want to be involved in Zenon. I try to keep a similar focus on friendship and merriment in my own actions." -- rex
And sugoi and bagwing wanted to add their submissions too. Bagwings submission: "This coin is not real." 
Sugoi's submission is "Being in the ZNN community has been an incredible journey so far, and I believe that the impact of NoM will be far greater than anyone could ever expect. Don't fade the momentum anon, keep stacking zats." - Sugoi

I'm happy for you guys to take over here, as I barely managed to make my original ghetto PR. Also I didn't know the twitter algorithm likes pictures, and I didn't consider the wisdom in tagging people as a form of fact-checking because it allows the tagged person to respond if needed. Please feel free to share more tips, they were very helpful and I learned a lot!

**Comment by sol-znn** [comment]
Glad to see the collaboration here! My tweets are in [this thread](https://twitter.com/SultanOfStaking/status/1687255069140353025).

**Comment by coinselor** [comment]
Great, @zyler9985, we'll incorporate the missing quotes and quote-tweet when possible, as well as tidy up anything else. 

As far as I understand, only maintainers can directly edit the PR. I could just send the final code to 0x and have him edit the PR. However, it would be nice if there was a setting to allow reviewers to suggest code changes:

https://haacked.com/archive/2019/06/03/suggested-changes/

image


I can add comments to lines of codes but I'm missing that code suggestion icon. Perhaps @0x3639 or anyone else knows if that could be a possibility moving forward.

**Comment by zyler9985** [comment]
just incase you have trouble finding them, to tag them on twitter, here are ppl's twitter handles in order of the way i submitted in PR (which was somewhat arbitrary):
** note, would need to re-check the character limit by previewing since tagging the handle sometimes takes more characters
@coinselor  (but should be quote retweeted)
@DCZenon
@zashounet
@crypto_publius
@znnayiid (this is layiid)
DogWithStyle is unknown?
@0x3639
@shaimo 
Sigli – unknown/prefers their privacy?
@sol_sanctum (but should be quote retweeted)
@TrevorLeahy3 (but should be quote retweeted)
@NeoShredder (this is rainbowginger)
CEO of wall st is unknown?
@waalkerte (rex)
@cryptocheshire (shazz)
@SultanOfStaking
@zyler9985
@dodo_bait (tapwoot)
@dat_she_pepe (chadassa)
@lilbagwing
@sugoibtc

**Comment by mehowz** [comment]
> 

Does Twitter still impose the 25 tweet limit on threads (on creation)?

**Comment by mehowz** [comment]
The way I'd do this:

1) Find the new Twitter algorithm.

2) Using the GPT4 plugin named "WebPilot" I'd show GPT a link to the Twitter algo so it can study the code.

3) Using the GPT4 plugin named "WebPilot" I'd show GPT the[ twitter-together documentation](https://github.com/marketplace/actions/twitter-together) so it can study it the formatting permitted.

4) I'd then use that GPT thread to compose my Tweets and have GPT not only format the tweet properly for twitter-together, but also optimize it for the new Twitter algo. Over time the GPT thread can relearn the code if it gets updated.

You can then ask GPT questions like:

1) Should I put images? If yes how many and where?

2) Should I ask the testimonials to be posted by the authors to their own accounts, and then be quoted rather than rewritten?

3) Should I use hashtags, if yes how many and where?

Essential I would build myself tools.

As far as content type, the archetype for the brand to use, audience strategy etc -- that's a whole different topic, though remember that GPT is your friend. Since I know that Twitter may not be friendly to external links, I'd think the objective would be to have readers visit the profile bio and click on the only call to action which'll be the website link. That's essentially the start of the funnel.

IMO the zenon_network handle should be used to:

1) Post general updates/milestones about the network.

2) Engage in a relevant way in conversations in the outbound to create awareness. Be more useful than eager to post links and have call-to-actions. It needs to feel natural and organic. It needs to develop trust.

3) Retweet ecosystem tweets.

**Comment by coinselor** [comment]
This is a WIP. I've refactored a lot of the initial text in favor of a cleaner thread. The content will be included in the first quote retweet. Anything I couldn't quote retweet was left out, if we want to include any specific content, we can ask someone to make a tweet about it.

TODO:
- [x] Featured image tribute.jpg + alt text
- [x] Add first quote retweet status ID
- [x] Add 1-2 hashtags
- [x] Include a few of the people I left out back in / Reach out to them for a tweet i.e Shaimo/Tapwoot/etc

I should probably set up my own twitter, together for testing, but since @Sol-Sanctum / @0x3639 already have it setup, hopefully one of them can answer these questions:

- Do line breaks work ?
- To properly format the thread/chain, should we do:

```
---
retweet: URL
---
---
---
retweet: URL
---
---
``` 

It's confusing because the threadDelimiter is the same as the syntax for the frontmatter. We should consider changing it. 



~~WORK-IN-PROGRESS~~ **DONE** 
```
---
media:
  - file: tribute.png
    alt: An alien spaceship beaming down the keys to the Zenon Network
---

A testament to the profound shift in power, Zenon’s transformation to a fully community-led project marks a milestone in the journey of open-source ethos.

To commemorate this moment, here’s a tribute thread composed entirely of quotes from community members.
---

Leaderless.

A concept that has puzzled, perplexed and often alienated since inception. It has been the echo in the vast expanses of our shared digital universe, a word whispered by critics, cynics and doubters alike.
---

27th of July, 2023.

Mr. Kaine airdropped the keys for Twitter/X and Github to the Zenon Community.

Now, more than ever, #TheAliensKnow
---
retweet: https://twitter.com/coinselor/status/1709697437290414305
---
retweet: https://twitter.com/mehowbrainz/status/1506072440283222019
---
retweet: https://twitter.com/DCZenon/status/1687266770951061505
---
retweet: https://twitter.com/zashounet/status/1688995460063657984
---
retweet: https://twitter.com/crypto_publius/status/1419503629778620416
---
retweet: https://twitter.com/0x3639/status/1687276021131739137
---
retweet: https://twitter.com/sol_sanctum/status/1687298335869132800
---
retweet: https://twitter.com/NeoShredder/status/1687281745262182400
---
retweet: https://twitter.com/zyler9985/status/1687415847646875649
---
retweet: https://twitter.com/theprofessorbtc/status/1687342069541421056
---
retweet: https://twitter.com/TrevorLeahy3/status/1689240183277830144
---
retweet: https://twitter.com/cryptocheshire/status/1688971273450229769
---
retweet: https://twitter.com/sugoibtc/status/1701306910614438145
---
retweet: https://twitter.com/ZenonOrg/status/1599650089659953153
---
retweet: https://twitter.com/mehowbrainz/status/1645985907713908736
---
```

**Comment by sol-znn** [comment]
The WIP looks like it's formatted correctly, and I believe the line breaks will display as expected.

I'm not sure how the last delimitter will be handled.

I can test this on my account tomorrow.

**Comment by mehowz** [comment]
And missing hashtags within the content, as suggested here: https://forum.zenon.org/t/develop-method-for-twitter-together/1610/5?u=mehowbrainz

**Comment by mehowz** [comment]
@0x3639 do you have permissions to create a project in this repository called "Tweet Calendar"? Namely a "Roadmap" type (as seen below) so that we can collaborate on a calendar of tweets? Would be awesome if we can get a way to queue tweets for posting (schedule them) using twitter-together. This way we just keep filling the queue.

Screen Shot 2023-09-14 at 10 44 34 AM

**Comment by zyler9985** [comment]
@coinselor we have some more testimonies rolling in! ill paste links for 4 new additions below from sugoi, zenonORG and two from mehowz:

https://twitter.com/sugoibtc/status/1701306910614438145?fbclid=IwAR2A9Ayh9M7qytBaWYToA4fZ2Y0y7oMVZKthPVvPF0gfrU3NrvhEGhiC2xw

https://twitter.com/ZenonOrg/status/1599650089659953153

https://twitter.com/mehowbrainz/status/1506072440283222019?s=20

https://twitter.com/mehowbrainz/status/1645985907713908736

**Comment by coinselor** [comment]
I've updated the WIP with the missing pieces, it's now completed. 

All we need to do is edit the original commit and add the `tribute.png` image file to the `media` directory. I think only @0x3639 has the capabilities to do so. Alternatively, we could create another PR, but I rather amend this one and get it over the finish line. Teamwork makes the dream work.

Thanks @zyler9985 for taking the initiative to submit the PR.

tribute

**Comment by 0x3639** [comment]
> I've updated the WIP with the missing pieces, it's now completed.
> 
> All we need to do is edit the original commit and add the `tribute.png` image file to the `media` directory. I think only @0x3639 has the capabilities to do so. Alternatively, we could create another PR, but I rather amend this one and get it over the finish line. Teamwork makes the dream work.
> 
> Thanks @zyler9985 for taking the initiative to submit the PR.
> 
> tribute

Actually I think @zyler9985 needs to resync  his master repo, that will pull in the changes that added the media folder.  Then he needs to resync his branch with his master.  That will add the /media folder to his branch.  then he can add the image to the /media in his branch.  

I cannot add an image directly to the media folder without overriding checks and balances, so I would like to avoid that.

**Comment by 0x3639** [comment]
OK just tried to fix this for @zyler9985. I uploaded the media file to the master media directory.  Apparently, the media file needs to be in the media directory of the PR.  I cannot do that for @zyler9985.  

Also, I had to override the permissions (2 reviewers) to delete this media file after I uploaded it to test.  I change the code of the tweet to add the media file.  Zyler just needs to rebase, and add the media.

**Comment by 0x3639** [comment]
Also, I can do this PR again from my account and just copy / paste the updated code.  We can all review again.  That might be easier and I'm happy to do that.

**Comment by sol-znn** [comment]
This can be closed. Well done, everyone!

https://twitter.com/Zenon_Network/status/1712141659373285436

**Pull request closed without merging**